moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp

```diff
@@ -59,29 +59,29 @@ class LinearPathSegment : public PathSegment
   {
   }
 
-  Eigen::VectorXd getConfig(double s) const
+  Eigen::VectorXd getConfig(double s) const override
```

Comment:
I'm curious why these `override` issues were not caught by Travis.

Reply:
I'm sure that clang-tidy related pkgs were cleaned up and reinstalled as it is in the Travis. But I'm not sure about the difference of the result, I will dig into that.